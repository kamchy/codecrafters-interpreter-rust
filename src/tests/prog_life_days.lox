var a = 1978;
var b = 2024;
var days = (b - a + 1) * 365;
print "Number of days in my life:"
print days;