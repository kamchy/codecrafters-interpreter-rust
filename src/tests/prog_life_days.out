Number of days in my life:
17155